src/components/rangeUsePlanPage/index.js

```diff
@@ -72,13 +72,42 @@ const Base = ({
   const getPlanId = () =>
     match.params.planId || location.pathname.charAt('/range-use-plan/'.length)
 
+  // for plans extending past agreement date, extend usage
+  const appendUsage = (plan, Plan) => {
+    let planStartDate = new Date(plan.planStartDate)
+    let planEndDate = new Date(plan.planEndDate)
+    let agrEndDate = new Date(plan.agreement.agreementEndDate)
+
+    var newPlan = JSON.parse(JSON.stringify(plan))
+
+    if (planEndDate.getFullYear() > agrEndDate.getFullYear()) {
+      let lastYearOfUsage =
+        plan.agreement.usage[plan.agreement.usage.length - 1]
+      var lastYearOfUsageID = lastYearOfUsage.id
+      var lastYear = lastYearOfUsage.year
+      do {
+        lastYearOfUsageID++
+        lastYear++
+        let tempUsage = JSON.parse(JSON.stringify(lastYearOfUsage))
+        tempUsage.id = lastYearOfUsageID
```

Comment:
We may also want to add an additional `fta: false` value to the appended usage objects for later use.

Reply:
done